docs/fhir/fhir_import_config.rst

```diff
@@ -65,7 +65,7 @@ and select the case type.
     to send the same resources back to FHIR when they are modified in
     CommCare, then you will either need to stick to the Data Dictionary
     FHIR resource types limitation, or add the resource type you want to
-    the list in `corehq/motech/fhir/const.py`_.)
+    the ``FHIR_DATA_TYPES`` const in ``corehq/motech/fhir/const.py``
```

Comment:
This _is_ the right constant, isn't it, @kaapstorm ?

Reply:
Oh! Good catch, but no. There was a time when we restricted the FHIR resource types (e.g. "Patient", "Observation", "Referral", etc.) that we integrated with, using a list in `const.py`. Since these docs were written, we dropped that list (and forgot to update these docs :face_with_diagonal_mouth: ). We allow anything that we have a schema for. All of the schemas can be found in `corehq/motech/fhir/json-schema/r4/`.

Reply:
:thinking: It might be best just to drop this `note` section. Using the Data Dictionary for configuring FHIR integration was always hamstringed, because it puts unrealistic limitations on mappings. I think any real integration will always need to use the Admin interface for some mappings. I would love to drop the FHIR-related code from Data Dictionary.